docs/about.md

```diff
@@ -25,12 +25,12 @@ Select one of the content blocks below to begin your Pulsar journey. If you ...
 * An experienced Pulsar coder looking for API, metrics, or configuration documentation, go to **Reference**. 
 
 <BlockLinks>
-    <BlockLink title="About Pulsar" url="/docs/next/concepts-overview/" />
-    <BlockLink title="Get Started" url="/docs/next/getting-started-home/" />
-    <BlockLink title="Install, Deploy, Upgrade" url="/docs/next/install-deploy-upgrade-landing/" />
-    <BlockLink title="Pulsar for Developers" url="/docs/next/developers-landing/" />
-    <BlockLink title="How To" url="/docs/next/how-to-landing/" />
-    <BlockLink title="Reference" url="/docs/next/reference-landing/" />
+    <BlockLink title="About Pulsar" url="concepts-overview" />
+    <BlockLink title="Get Started" url="/docs/next/getting-started-home" />
+    <BlockLink title="Install, Deploy, Upgrade" url="/docs/next/install-deploy-upgrade-landing" />
+    <BlockLink title="Pulsar for Developers" url="/docs/next/developers-landing" />
+    <BlockLink title="How To" url="/docs/next/how-to-landing" />
```

Comment:
Shall we apply the relative path for these tags consistently also?

Reply:
You are right! Thanks for finding out more occurrences that I missed.